client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -208,38 +221,84 @@ spec:
             description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           spec:
-            description: spec defines properties of a VolumeSnapshotContent created by the underlying storage system. Required.
+            description: spec defines properties of a VolumeSnapshotContent created
```

Comment:
Can you keep the API changes only to v1 and leave v1beta1 the same?

Reply:
Even the indentation changes?

Reply:
Yes

Reply:
Regarding this particular line, it also looks like you only changed the v1beta1 section but not the v1 section.
Please leave v1beta1 section unchanged, including indentation changes.

What you can do is to run the script first, then copy the original v1beta1 section over.

Reply:
Done